macros/edr/metadata_collection/get_columns_from_information_schema.sql

```diff
@@ -2,20 +2,40 @@
     {{ return(adapter.dispatch('get_columns_from_information_schema', 'elementary')(full_schema_name)) }}
 {% endmacro %}
 
-{# Snowflake and Bigquery #}
+{# Snowflake, Bigquery#}
 {% macro default__get_columns_from_information_schema(full_schema_name) %}
     {% set full_schema_name_split = full_schema_name.split('.') %}
     {% set database_name = full_schema_name_split[0] %}
     {% set schema_name = full_schema_name_split[1] %}
+    {% set schema_name_string = schema_name | replace('"','') %}
```

Comment:
Instead of doing it - change the get_configured_schemas_from_graph to return list of [db, schema] pairs